tensilelite/Tensile/KernelWriter.py

```diff
@@ -5281,3 +5072,11 @@ def findInstByName(module, names, count):
         else:
           _placeholder.add(SBranch(labelName=_target.getLabelName()))
       currentInstLength += _placeholder.countType(Instruction)
+
+  @property
+  def isa(self):
+    return self.states.version
+
+  @property
+  def wavefrontSize(self):
+    return self.states.kernel["WavefrontSize"]
```

Comment:
I hit an error on this line in testing:

  File "Tensile/TensileCreateLibrary.py", line 101, in processKernelSource
    wavefrontSize = kernelWriter.wavefrontSize
  File "Tensile/KernelWriter.py", line 5082, in wavefrontSize
    return self.states.kernel["WavefrontSize"]
KeyError: 'WavefrontSize'